src/pages/Scholarships.jsx

```diff
@@ -26,14 +29,41 @@ function ScholarshipsPage() {
   const [error, setError] = useState();
   const [sortField, setSortField] = useState('deadline');
   const [sortDir, setSortDir] = useState('asc');
+  const [amountFilterVals, setAmountFilterVals] = useState({ min: 0, max: 0 });
+  const location = useLocation();
+
+  useEffect(() => {
+    const params = queryString.parse(location.search);
+    const minParam = parseInt(params.min ?? 0, 10);
+    const maxParam = parseInt(params.max ?? 0, 10);
+    setAmountFilterVals({
```

Comment:
I tried pruning the invalid URL search  parameters after this by doing
 `let params = new URLSearchParams(location.search);
params.delete('min')` but it didn't work and I noticed the URLSearchParams returned a null object.

Reply:
ooo yes it worked!